hack/ds.yaml

```diff
@@ -111,3 +111,14 @@ spec:
     enabled: false
     path: gs://ds-operator-engineering-devops/ds-backup-test
     secretName: cloud-storage-credentials
+
+  #### Multi-region ####
+  # multiRegion:
+    # clusterTopology and clusterRegion can be used for multiple multi-region solutions specifically the
+    # kubedns and MCS solutions documented in forgeops.
+    # clusterTopology - list of region identifiers for the each cluster used to determine the bootstrap servers. 
+    # clusterTopology: "eu,us"
+    # clusterRegion - current clusters region identifier.  Must match 1 of the identifiers in clusterTopology
+    # clusterRegion: "us"
+    # Enable MCS(Googles Multi-cluster Services).  
```

Comment:
Its not just Google ? (should be a K8S standard).

Reply:
Well it is classed as a GKE thing because it interacts with GCP specific services.  But I know they want to adapt it for other Clouds.  I'll remove 'Google'

Reply:
If it's not google, then let's specify a cloud provider somewhere. 
```
provider: GKE | AWS | etc
mcsEnabled: true
```

Reply:
It is a GKE service.   I think what Warrens implying is the in the future they want to extend it to span cloud providers.   But I'm not sure whether that will come under the same name.   
Quote: `MCS is a cross-cluster Service discovery and invocation mechanism for Google Kubernetes Engine (GKE) that leverages the existing Service object`

Reply:
I think we can add other providers in the future if and once we know how this will evolve.  May not happen for a long time